docs/content/preview/reference/configuration/insecure-yugabyted-examples.md

```diff
@@ -0,0 +1,939 @@
+```
+
+This command determines the data placement constraint based on the `--cloud_location` of each node in the cluster. If there are 3 or more regions available in the cluster, the command configures the cluster to survive at least 1 region failure. Otherwise, it outputs a warning message.
+
+The replication factor of the cluster defaults to 3.
+
+You can set the data placement constraint manually using the `--constraint_value` flag, which takes the comma-separated value of `cloud.region.zone` as follows:
+
+```sh
+./bin/yugabyted configure data_placement --fault_tolerance=region --constraint_value=aws.us-east.us-east-1a,aws.us-west.us-west-1a,aws.us-central.us-central-1a
+```
+
+You can set the replication factor of the cluster manually using the `--rf` flag as follows:
+
+```sh
+./bin/yugabyted configure data_placement --fault_tolerance=region --constraint_value=aws.us-east.us-east-1a,aws.us-west.us-west-1a,aws.us-central.us-central-1a --rf=3
+```
+
+### Enabling and Disabling encryption at rest
+
+Create a cluster using the desired deployment example.
+
+To enable encryption at rest in a deployed local cluster, run the following command:
+```sh
+./bin/yugabyted configure encrypt_at_rest --enable --base_dir=$HOME/yugabyte-{{< yb-version version="preview" >}}/node1
+```
+
+To enable encryption at rest in a deployed multi-zone or multi-region cluster, run the following command from any VM:
+```sh
+./bin/yugabyted configure encrypt_at_rest --enable
+```
+
+To disable encryption at rest in a local cluster with encryption at rest enabled, run the following command:
+```sh
+./bin/yugabyted configure encrypt_at_rest --disable --base_dir=$HOME/yugabyte-{{< yb-version version="preview" >}}/node1
+```
+
+To disbale encryption-at-rest in a multi-zone or multi-region cluster with encryption-at-rest enabled, run the following command from any VM:
+```sh
+./bin/yugabyted configure encrypt_at_rest --disable
+```
+
+### Pass additional flags to YB-TServer
+
+Create a single-node cluster and set additional flags for the YB-TServer process.
+
+```sh
+./bin/yugabyted start --tserver_flags="pg_yb_session_timeout_ms=1200000,ysql_max_connections=400"
+```
+
+## Upgrade a YugabyteDB cluster
+
+To use the latest features of the database and apply the latest security fixes, it's prudent to upgrade your YugabyteDB cluster to the [latest release](https://download.yugabyte.com/).
+
+Upgrading an existing YugabyteDB cluster that was deployed using yugabyted includes the following steps:
+
+1. Stop the running YugabyteDB node using the `yugabyted stop` command.
+
+1. Start the new yugabyted process by executing the `yugabyted start` command. Use the previously configured `--base_dir` when restarting the instance.
+
+Repeat the steps on all the nodes of the cluster, one node at a time.
+
+### Upgrade a cluster from single to multi zone
+
+{{< note title="Note" >}}
+
+Multi-zone, multi-region deployment using yugabyted is supported in YugabyteDB 2.15.0.0 and later.
```

Comment:
The note is not needed, because this document will only be published for YDB version 2.15 and later. The readers of earlier release docs will not see it.